docs/book/src/development/releasing.md

```diff
@@ -2,18 +2,32 @@
 
 ## Manual
 
-1. Make sure your repo is clean by git's standards
-2. Set environment variable `GITHUB_TOKEN` to a GitHub personal access token
-3. If this is a new minor release, create a new release branch and push to GitHub, otherwise switch to it, for example `release-0.6`
-4. Tag the repository and push the tag `git tag -s -m $VERSION $VERSION`. `-s` flag is for GNU Privacy Guard (GPG) signing
-5. Push the commit of the tag to the release branch: `git push origin HEAD:release-0.6`
-6. Set environment variables `PREVIOUS_VERSION` which is the last release tag and `VERSION` which is the current release version.
-7. Checkout the tag you've just created and make sure git is in a clean state
-8. Export the current branch `BRANCH=release-0.6` (`BRANCH=main`)and run `make release`
-9. A prow job will start running to push images to the staging repo, can be seen [here](https://testgrid.k8s.io/sig-cluster-lifecycle-image-pushes#post-cluster-api-provider-aws-push-images).
-10. Run `make create-gh-release` to create a draft release on Github, copying the generated release notes from `out/CHANGELOG.md` into the draft.
-11. Run `make upload-gh-artifacts` to upload artifacts from .out/ directory, however you may run into API limit errors, so verify artifacts at next step
-12. Verify that all the files below are attached to the drafted release:
+1. Make sure all [periodic tests](https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-aws) are passing.
```

Comment:
As a side-note, I would just use
```
1.
1.
1.
```

Because then Markdown auto-increases the numbers upon rendering. So you don't have to rearrange these step numbers each time you insert/remove a step. :)

Reply:
Yeah, good point, noticed that somewhere but didn't go and check if this actually works.